tests/serialized_test_data_generation/Makefile

```diff
@@ -29,7 +29,7 @@ setup_rundir:
 	git rev-parse HEAD > $(RUNDIR_HOST)/fortran_sha.txt ; fi
 
 compile_fortran:
-	cd $(FORTRAN_DIR) && DOCKER_BUILDKIT=1 SERIALIZE_IMAGE=$(COMPILED_IMAGE) $(MAKE) build_serialize_gt4pydev
+	cd $(FORTRAN_DIR) && DOCKER_BUILDKIT=1 SERIALIZE_IMAGE=$(COMPILED_IMAGE) $(MAKE) build_serialize
```

Comment:
Already indicated above, but this  renaming obscures the fact that we are using the GT4PY_DEV compile flag, and I think it makes to emphasize it for this.

Reply:
Yes, I agree with @rheacangeo  that the older convention is more clear.

Reply:
See above. Name change undone.